pkgs/development/python-modules/python-hcl2/default.nix

```
@@ -0,0 +1,35 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  pythonOlder,
+  lark,
+  setuptools_scm,
+}:
+
+buildPythonPackage rec {
+  pname = "python-hcl2";
+  version = "4.3.4";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-7xubrQGLz8H+J5IESXQpnlWRRf6W48opjB5elQDI3mY=";
+  };
+
+  disabled = pythonOlder "3.7";
+
+  build-system = [ setuptools_scm ];
+
+  dependencies = [ lark ];
+
+  pythonImportsCheck = [ "hcl2" ];
+
```

```suggestion
  nativeCheckInputs = [ pytestCheckHook ];
```
Could you enable a check phase with `pytestCheckHook`?
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pytestcheckhook-using-pytestcheckhook